resources/tags.yaml

```diff
@@ -0,0 +1,129 @@
+accounting:
+  title: 'Accounting'
+  description: 'Processing and communication of financial information'
+analytics:
+  title: 'Analytics'
+  description: 'Data or statistics to make intelligent marketing and business decisions'
+backend: 
+  title: 'Backend'
+  description: 'Backend'
+blogging: 
+  title: 'Blogging'
+  description: 'Informational site published with discrete entries'
+calendars:
+  title: 'Calendars'
+  description: 'Calendars'
+cloud_platform: 
+  title: 'Cloud Platform'
+  description: 'Offers hosting on the same supporting infrastructure'
+cms: 
```

Comment:
`cms` is too specific, it should be more generic.
Like `frontend`,`web sites` or something similar.